Sources/TuistSupport/Utils/Glob.swift

```diff
@@ -113,15 +113,37 @@ public class Glob: Collection {
     }
 
     private func expandGlobstar(pattern: String) -> [String] {
-        guard pattern.contains("**") else {
+        // Split pattern string by slash to find globstar.
+        let patternComponents = pattern.components(separatedBy: "/")
+
+        // We are only interested in the first globstar since that is where we want to separate the pattern string.
+        guard let pivot = patternComponents.firstIndex(of: "**") else {
```

Comment:
The [implementation](https://gist.github.com/efirestone/ce01ae109e08772647eb061b3bb387c3) you referenced in tests expands arbitrary number of globstars. Since we're touching this utility, I wonder if we should add support for that, but we can also keep the current behavior

Reply:
sorry, I missed that 🙏